rpc/core/blocks.go

```diff
@@ -75,7 +75,8 @@ func filterMinMax(base, height, min, max, limit int64) (int64, int64, error) {
 	min = tmmath.MaxInt64(min, max-limit+1)
 
 	if min > max {
-		return min, max, fmt.Errorf("min height %d can't be greater than max height %d", min, max)
+		return min, max, fmt.Errorf("%w: min height %d can't be greater than max height %d",
```

Comment:
Typically, the error is wrapped at the end (e.g. `fmt.Errorf("error message...: %w", err)`)

Reply:
Right but my goal is to wrap this min/max height error with a more general invalid request error so that when it comes to sending the response I can see that this error belongs to the category of invalid request and I can return the appropriate jsonrpc code

Reply:
I suppose if the wrapping and unwrapping works as expected, this is fine.